plutus-ledger-api/exe/dump-cost-model-parameters/Parsers.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Parsers (Format(..), WhichLL(..), parseDumpOptions)
+where
+
+import Options.Applicative
+import PlutusLedgerApi.Common.Versions (PlutusLedgerLanguage (..))
+
+data WhichLL =
+    One PlutusLedgerLanguage   -- Print parameters for a single LL.
+  | All                        -- Print parameters for all LLs.
+  deriving stock (Show)
+
+parseVersion :: ReadM WhichLL
+parseVersion = eitherReader $ \case
+  "1" -> Right $ One PlutusV1
+  "2" -> Right $ One PlutusV2
+  "3" -> Right $ One PlutusV3
+  s -> Left $ "Unknown ledger language version: " ++ s
+
+whichll :: Parser WhichLL
+whichll =
```

Comment:
The indentation is inconsistent across newly added files,
do you mind applying Fourmolu to all the Haskell code in this PR?

Reply:
> do you mind applying Fourmolu to all the Haskell code in this PR?

I've done that, but then stylish-haskell changed some of it back when I committed the changes.  Specifically, it lined up the `->`s in case expressions and also the `-#}` at the end of the pragmas at the start.  I like both of those better than what Fourmolu does, but you might object!

Reply:
I'm not sure but I think that also Fourmolu aligned `let ... in ...` like this
```
  let xxx
   in yyy
```
and stylish-haskell changed it back to
```
  let xxx
  in yyy
```
The first one looks a bit strange to me but I see why it's done it.  It's a pity that `let` and `in` don't have the same number of letters...